src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
add field `ColorSupplier colorSupplier` instead of extending 
if you use `extends` the relationship between  `Lottery` and `ColorSupplier` should be **is a**
please check in which case can be used inheritance